extensions/visualizations/OppiaVisualizationFrequencyTableDirective.js

```diff
@@ -13,7 +13,7 @@
 // limitations under the License.
 
 /**
- * @fileoverview Directives for all reusable data visualization components.
+ * @fileoverview Directive for visualization frequency table.
```

Comment:
Directive for the "frequency table" visualization.

Reply:
done